packages/babel-plugin-ember-test-metadata/utils.js

```diff
@@ -1,4 +1,5 @@
 const path = require('path');
+const EMBROIDER = 'embroider';
```

Comment:
Why do we need this to be a const?

Reply:
This string is used in 2 different places.

Reply:
What's the problem with having a string in two places? It's more readable to inline this, and I'm not a fan of creating string constants just because they're used in more than one place except if they're of sufficient complexity that they'd need to be done once to avoid bugs. In this case I'm not sure it fits that.